libcodechecker/libhandlers/check.py

```diff
@@ -298,6 +298,49 @@ def add_arguments_to_parser(parser):
                                     "'<OUTPUT_DIR>/ctu-dir'. (These files "
                                     "will not be cleaned up in this mode.)")
 
+    if host_check.is_statistics_capable():
```

Comment:
We duplicate huge number of code in the `CodeChecker analyze` and `CodeChecker check` command. Wouldn't it be better to put these codes to a common place? It would be better maintainable.
https://github.com/Ericsson/codechecker/blob/830dd165d8ea807447b9552d348cee99d887eddc/libcodechecker/libhandlers/analyze.py#L237-L377
What do you think @gyorb?

Reply:
It was a design decision earlier to duplicate the code in this case.